app/models/survey_notification.rb

```diff
@@ -39,4 +40,24 @@ def user
   def course
     Course.find(course_id)
   end
+
+  private
+
+  MAX_FOLLOW_UPS = 3
+  def ready_for_follow_up?
+    return false if email_sent_at.nil?
+    return false if Time.now < last_email_sent_at + time_between_emails
+    return false if follow_up_count >= MAX_FOLLOW_UPS
+    true
+  end
+
+  def time_between_emails
```

Comment:
I find that really confusing. Maybe `time_between_intial_and_most_recent_follow_up`?